src/routes/SearchRoute/SearchRoute.test.js

```diff
@@ -1,10 +1,31 @@
-import { render } from '@testing-library/react';
+import {
+  fireEvent,
+  render,
+} from '@testing-library/react';
 
 import SearchRoute from './SearchRoute';
 import Harness from '../../../test/jest/helpers/harness';
 
+const mockOnSubmitSearch = jest.fn();
+
 jest.mock('../../views', () => ({
-  AuthoritiesSearch: ({ children }) => <div>AuthoritiesSearch <div>{children}</div></div>,
+  AuthoritiesSearch:
+  ({
+    children,
+  }) => (
```

Comment:
This can be left as it was - in one line
```suggestion
  ({ children }) => (
```

Reply:
You are right, thanks!